pom.xml

```
@@ -167,6 +167,12 @@
             <version>1.11</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>pipeline-build-step</artifactId>
+            <version>2.10-rc366.8d263648aa27</version> <!-- TODO: Pending release of https://github.com/jenkinsci/pipeline-build-step-plugin/pull/24 -->
```

@basil I just released pipeline-build-step.

```suggestion
            <version>2.10</version>
```